registrations/tasks.py

```diff
@@ -80,6 +80,80 @@ def calc_baby_age(today, baby_dob):
         return -1
 
 
+def get_messageset_short_name(stage, recipient, msg_type, weeks):
+
+    if recipient == "household":
+        msg_type = "text"
+
+    if stage == "prebirth":
+        week_range = "10_42"
+    elif stage == "postbirth":
+        if 0 <= weeks <= 12:
+            week_range = "0_12"
+        elif 13 <= weeks <= 52:
+            week_range = "13_52"
+    elif stage == "loss":
+        week_range = "0_2"
+
+    short_name = "%s_%s_%s_%s" % (
+        stage, recipient, msg_type, week_range)
+
+    return short_name
+
+
+def get_cron_string(days, times):
+    t1 = "0"
+    t2_map = {
+        '9_11': "8",
+        '2_5': "13"
+    }
+    t2 = t2_map[times]
+    t3_map = {
+        'mon_wed': "1,3",
+        'tue_thu': "2,4"
+    }
+    t3 = t3_map[days]
+    t4 = "*"
+    t5 = "*"
+
+    return "%s %s %s %s %s" % (t1, t2, t3, t4, t5)
+
+
+def get_messageset_schedule_sequence(short_name, weeks, voice_days,
+                                     voice_times):
+    # get messageset_id
+    url = settings.MESSAGESET_URL
+    params = {'short_name': short_name}
+    headers = {'Authorization': ['Token %s' % settings.MESSAGESET_TOKEN],
+               'Content-Type': ['application/json']}
+    r = requests.get(url, params=params, headers=headers)
+    messageset_id = r.json()["id"]
```

Comment:
these services calls need to be abstracted into a function that is using a client that handles retries and failures etc. For now, open an issue for the client and just move to function to unblock this issue.